### PR TITLE
feat(#1575): All classes are compiled test

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I've added disabled integration test that checks all classes that had to be compiled. We will be able to enable it after #1576

Closes: #1575

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you take a look, please?

File: eo-runtime/pom.xml [hunk 1]
```diff
@@ -144,7 +148,8 @@ SOFTWARE.
               <targetDir>${project.basedir}/target/eo-test</targetDir>
               <addSourcesRoot>false</addSourcesRoot>
               <addTestSourcesRoot>true</addTestSourcesRoot>
-              <generatedDir>${project.basedir}/target/generated-test-sources</generatedDir>
+              <generatedDir>${project.basedir}/target/generated-test-sources
+              </generatedDir>
```
Comment (yegor256) [inline eo-runtime/pom.xml:152]: @volodya-lombrozo it's a typo?

File: eo-runtime/pom.xml [hunk 2]
```diff
@@ -95,9 +95,13 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>@{argLine} -Xss${stack-size} -Duser.language=ru -Duser.region=RU</argLine>
+          <argLine>@{argLine} -Xss${stack-size} -Duser.language=ru
+            -Duser.region=RU
```
Comment (yegor256) [inline eo-runtime/pom.xml:99]: @volodya-lombrozo also here, looks a bit wrong to me. Why not all in one line?

File: eo-runtime/pom.xml (hunk 1 above)
Comment (volodya-lombrozo) [reply eo-runtime/pom.xml:152]: @yegor256 It was `xcop --fix`. I'll return it back

File: eo-runtime/pom.xml (hunk 2 above)
Comment (volodya-lombrozo) [reply eo-runtime/pom.xml]: @yegor256 fixed

File: eo-runtime/pom.xml (hunk 1 above)
Comment (volodya-lombrozo) [reply eo-runtime/pom.xml:153]: @yegor256 fixed

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**